lib/pbench/test/unit/server/test_user_create.py

```diff
@@ -132,54 +205,53 @@ def test_valid_users_update(client, server_config, pytestconfig):
-                user_create,
+                user_update,
                 args=[
-                    TestUserManagement.USER_SWITCH,
                     TestUserManagement.USER_TEXT,
-                    TestUserManagement.EMAIL_SWITCH,
-                    TestUserManagement.EMAIL_TEXT,
-                    TestUserManagement.FIRST_NAME_SWITCH,
-                    TestUserManagement.FIRST_NAME_TEXT,
-                    TestUserManagement.LAST_NAME_SWITCH,
-                    TestUserManagement.LAST_NAME_TEXT,
                     TestUserManagement.ROLE_SWITCH,
                     "ADMIN",
                 ],
-                input=f"{TestUserManagement.PSWD_TEXT}\n",
             )
             assert result.exit_code == 0
+            assert result.stdout == "User test_user updated\n"
+            assert not result.stderr_bytes
+
+            # Update invalid role of the user
+            result = runner.invoke(
+                user_update,
+                args=[
+                    TestUserManagement.USER_TEXT,
+                    TestUserManagement.ROLE_SWITCH,
+                    "ADMN",
+                ],
+            )
+            assert result.exit_code == 2
             assert (
-                result.stdout
-                == f"{TestUserManagement.PSWD_PROMPT}\n"
-                + "Admin user test_user registered\n"
+                result.stderr_bytes
+                == b"Usage: user-update [OPTIONS] UPDATEUSER\nTry 'user-update --help' for help.\n"
+                + b"\nError: Invalid value for '--role': invalid choice: ADMN. (choose from ADMIN)\n"
             )
-            assert not result.stderr_bytes
 
     @staticmethod
     @responses.activate
-    def test_user_creation_with_invalid_role(client, server_config, pytestconfig):
+    def test_invalid_user_update(client, server_config, pytestconfig):
         with client:
             runner = CliRunner(mix_stderr=False)
+
+            # Update with non-existent username
             result = runner.invoke(
-                user_create,
+                user_update,
                 args=[
-                    TestUserManagement.USER_SWITCH,
                     TestUserManagement.USER_TEXT,
                     TestUserManagement.EMAIL_SWITCH,
-                    TestUserManagement.EMAIL_TEXT,
-                    TestUserManagement.FIRST_NAME_SWITCH,
-                    TestUserManagement.FIRST_NAME_TEXT,
-                    TestUserManagement.LAST_NAME_SWITCH,
-                    TestUserManagement.LAST_NAME_TEXT,
-                    TestUserManagement.ROLE_SWITCH,
-                    "ADMN",
+                    "new_test@domain.com",
                 ],
-                input=f"{TestUserManagement.PSWD_TEXT}\n",
             )
-            assert result.exit_code == 2
+            assert result.exit_code == 1
+            assert (
+                result.stdout
+                == f"No such a user {TestUserManagement.USER_TEXT} to update\n"
```

Comment:
Apparently I missed this in the implementation, but "no such a user" isn't right. Also, maybe just "User {...} doesn't exist" or "No such user {...}" since the "update" is implicit in the context anyway.